app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -1,246 +1,252 @@
-h1 {
-    .lib-visually-hidden();
-}
-.section.section__slider {
-    padding-bottom: 30px;
-}
-
-#maincontent {
-    max-width: 100%;
-    padding: 0;
-    margin: 0;
-}
-
-.section {
-    padding-bottom: 30px;
-    h2,
-    h3 {
-        font-size: 24px;
-        font-weight: 600px;
-        letter-spacing: 1px;
-        text-transform: uppercase;
-        color: @color-maksym__main-black;
-    }
-
-    p {
-        font-size: 14px;
-        line-height: 22px;
-        font-weight: 300px;
-        letter-spacing: 1px;
-        color: @color-maksym__main-gray;
-    }
-    button {
-        font-size: 14px;
-        line-height: 22px;
-        font-weight: 300px;
-        letter-spacing: 1px;
-        color: @color-maksym__main-black;
-        &:hover,
-        &:focus {
-            color: inherit;
-        }
+& when (@media-common = true) {
+    h1 {
+        .lib-visually-hidden();
     }
-    &__title {
+    .section.section__slider {
+        padding-bottom: 30px;
     }
-}
 
-/* SECTION__SLIDER */
+    #maincontent {
+        max-width: 100%;
+        padding: 0;
+        margin: 0;
+    }
 
-.section__slider {
-    max-width: 1920px;
-    .slider {
-        &__title {
-            .lib-visually-hidden();
-        }
-        .pagebuilder-slide-wrapper {
-            display: flex;
-            align-items: center;
-            justify-content: flex-end;
+    .section {
+        padding-bottom: 30px;
+        h2,
+        h3 {
+            .lib-font-size(24);
+            font-weight: @font-weight__semibold;
+            letter-spacing: 1px;
+            text-transform: uppercase;
+            color: @color-maksym__main-black;
         }
-        .pagebuilder-overlay {
-            max-width: 50%;
-            margin-right: 50%;
+
+        p {
+            .lib-font-size(14);
+            line-height: 22px;
+            font-weight: @font-weight__light;
+            letter-spacing: 1px;
+            color: @color-maksym__main-gray;
+        }
+        button {
+            .lib-font-size(14);
+            line-height: 22px;
+            font-weight: @font-weight__light;
+            letter-spacing: 1px;
+            color: @color-maksym__main-black;
+            &:hover,
+            &:focus {
+                color: inherit;
+            }
         }
-        [data-element="content"] {
-            padding-left: 17px;
+        &__title {
         }
+    }
 
-        .slick-dots {
-            position: absolute;
-            bottom: 0;
-            button {
-                width: 10px;
-                height: 10px;
-                border-radius: 0;
-                border: 1px solid @color-maksym__main-black;
+    /* SECTION__SLIDER */
+
+    .section__slider {
+        max-width: 1920px;
+        .slider {
+            &__title {
+                .lib-visually-hidden();
+            }
+            .pagebuilder-slide-wrapper {
+                .lib-vendor-prefix-display(flex);
+                align-items: center;
```

Comment:
there is a lib for align-items:
`.lib-vendor-box-align()`

Reply:
done